testing/integration/kubernetes_agent_standalone_test.go

```diff
@@ -60,7 +62,11 @@ func TestKubernetesAgentStandaloneKustomize(t *testing.T) {
 		Local: false,
 		Sudo:  false,
 		OS: []define.OS{
-			{Type: define.Kubernetes},
+			{Type: define.Kubernetes, DockerImage: "docker.elastic.co/beats/elastic-agent"},
```

Comment:
I think it would be easier to maintain if the list of  all images Docker to test was part of the testing framework and only when a test needs to runs under specific ones define them explicitly. In simple terms a test that has `{Type: define.Kubernetes }` should be running under all Docker images.

Also maybe we want to exclude the repository part of the Docker image as this can change and go only by image name?!

Reply:
I didn't do that mainly because I didn't want to hard code the image names in the testing framework. By placing them directly into the tests it removed the need for me to hard-code anything.

`{Type: define.Kubernetes}` in the testing framework case would not auto run the test on every docker variant it would just pick the first one that is defined as supported. Only in the case of `mage integration:matrix` would it then perform the tests on all versions of the image.

I lean towards keeping it this way, let me know your opinion.